src/test/resources/files.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<files xmlns:dcterms="http://purl.org/dc/terms/">
```

Comment:
next week this file may not be valid anymore... then we need a schema declaration here. Also `dcterms:accessRights` will be changed (@lindareijnhoudt is this actually true?)